src/smk/tool/geomark/tool-geomark.js

```diff
@@ -109,14 +109,24 @@ include.module( 'tool-geomark', [
                 }
             } 
 
+            this.defaultTemplineStyle = undefined;
+            this.defaultHintlineStyle = undefined;
             this.changedActive( function () {
                 if ( self.active ) {
+                    if (!this.defaultTemplineStyle) {
+                        this.defaultTemplineStyle = JSON.parse(JSON.stringify(smk.$viewer.map.pm.getGlobalOptions().templineStyle));
```

Comment:
I'm using the JSON methods to capture the value of the style object - otherwise we only have a reference and we can't easily reset the style to the default. Let me know if you have better suggestions for capturing the value.

Reply:
This is great!